### ISSUE TITLE
Improve "System" system

### ISSUE THREAD

**Issue by jpcompton** [iftechfoundation/ifdb-suggestion-tracker#179]
The system field, as it has been implemented and evolved as a single entry, is an unholy mess of over 100 entries. Some are useless (just "Inform", for example, which tells one nothing, or "Dos"), others of very questionable utility (is the difference between HTML and HTML/CSS important?), still others redundant ("PAW" vs. "PAW (Professional Adventure Writer).")

And for the very corner cases where a search would be useful ("are there games coded in both Inform and {something else} that I could consult and learn from", for example) it's not, because there's no consistent enforcement about how those multi-system names are entered.

It feels like it rather wants to be a multi-select field, where a game could be

✅ Inform 7 ✅ Vorple

or

✅ Inform 6 ✅ PunyInform

rather than current blergy outcomes such as:

The Quill / Adventuron
The Quill / DAAD / Adventuron
The Quill, Adventuron, DAAD
The Quill, Inform 6
The Quill, Inform 6, ZIL
The Quill, PAW
The Quill, ZIL

**Comment by dfabulich** [comment]
Yeah, "System" is kind of a mess. The UI doesn't make it very clear, but it's actually a plain text field! You can type anything you want in there, even if it doesn't appear on the list. You can type "Ifnorm" as your system and IFDB will happily accept it.

**Comment by jpcompton** [comment]
Oh, I know! That's how we got here!

**Comment by curiousdannii** [comment]
It might be better to think of it like a tag system, and then make it possible to mark systems as duplicates of others.
